lib/puppet/functions/influxdb/retrieve_token.rb

```diff
@@ -6,30 +6,63 @@
     param 'String', :uri
     param 'String', :token_name
     param 'String', :admin_token_file
+    param 'Boolean', :use_system_store
+    optional_param 'String', :ca_bundle
```

Comment:
Can we use custom types from stdlib here? Could we switch to Stdib::Absolutepath?

Reply:
Would we have issues using this with `Deferred`?  I remember we hit something like that so I decided to keep it simple with a `String`

Reply:
I removed the `ca_bundle` option in favor of Jarret's suggestion to use `ssl_trust_store` in `puppet.conf`, so this no longer applies.